### DISCUSSION TITLE
New Destination Firebird

### DISCUSSION THREAD

**Discussion by MinuraPunchihewa** [airbytehq/airbyte#36092] [New Connector Request]
## Tell us about the new connector you’d like to have
I have been working on a project that is currently running a [Firebird](https://firebirdsql.org/) database on our backend. We want to able to ingest data from a variety of data sources into our database including other databases like MySQL and Postgres.

## Describe the context around this new connector
We are currently in the process of evaluating Firebird for use in some of our analytical projects (our Architect is hugely fond of open source technologies). This use case is not really to analyze a huge volume of data, hence why we have not looked at using a data warehouse. We are required to pull in some data every so often into this database to run some ad hoc reports. As a result, we have been exploring open source solutions available for setting up our pipelines.
We will be looking at running syncs every week or so.

## Describe the alternative you are considering or using
We have considered using Airflow to write some Python tasks in order to achieve this, however, life would be so much easier if an integration was available in Airbyte!

## Are you willing to submit a PR?
Yes, I would love to work on this!

**Comment by sajarin** [comment]
Hey @MinuraPunchihewa, 

Thanks for the connector suggestion. We'd be happy to review the PR if you decide to work on it. Let me know!

**Comment by MinuraPunchihewa** [comment]
Hey @sajarin,
Apologies, I know it has been a while. But, would it be OK if I raised a PR for this?

**Comment by diegonery** [comment]
Any updates, about that?

**Comment by Nim-saj** [comment]
Je suis également intéressé par ce connecteur.
Des nouvelles ?

[a comment by an automated account is not shown]
